learn/wavemaker-release-notes/v10-7-0.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "WaveMaker 10.7.0 - Release date: 09 May 2021"
+id: ""
+sidebar_label: "Release v10.7.0"
+---
+*WaveMaker released a new update version with some new features and bug fixes.*
+
+## New Features
+
+---
+
+### Work in Branches
+
+- Create branches for a project in WaveMaker Studio for hotfixes and features.
+- Develop, preview, and test applications.
+- Branch operations such as create, merge and delete are required to perform on the source controller, i.e., outside WaveMaker.
+- This feature is enabled on request for WMO Teams subscribed customers.
+
+### VCS Enhancements
+
+- Support for project grouping in VCS.  
+- Create a project from the existing VCS repository URL (empty).
+- Provision to display the repo URL for a project.
+- Default branch for any VCS repository.
+
+### NPM Artifacts
+
+- Follows standard front-end way to publish and make it available to global developers.
+- Easily install and explore in a trusted public server accessible 24/7.
+- No more the need for S3 storage.
+
+### Support for Red Shift Database
+
+- Connect to an existing database in AWS RedshiftDB from a WaveMaker application. 
+- Store petabytes of data in easy-to-access "clusters" that can be queried in parallel.
+
+### Support for SAP Hana Database
+
+- Connect to SAP Hana Cloud database and configure the relational database schema in WaveMaker.
+- Use In-Memory Database to store huge databases for analysis or information processing.
+
+### Allow Hosts
```

Comment:
I didn't understand this at all. @swethakundaram Collaborate with the concerned developer and write this better